libs/core/src/lib/button/button.component.ts

```diff
@@ -37,7 +37,8 @@ import { applyCssClass } from '@fundamental-ngx/core/utils';
     changeDetection: ChangeDetectionStrategy.OnPush,
     host: {
         '[attr.type]': 'type',
-        '[attr.disabled]': '_disabled || null'
+        '[attr.disabled]': '_disabled || null',
+        '[attr.aria-label]': 'ariaLabel + " " + fdType'
```

Comment:
i don't think this is needed. See above comment on `button-bar`. Also, this does not check for undefined `fdType` cases.
![image](https://user-images.githubusercontent.com/53509521/129673214-21f56b51-e9bd-4dbc-b127-f089643b1c6b.png)

Reply:
fixed